### PR TITLE
Docs: Combines thresholds docs into a single topic

### PR DESCRIPTION
**Pull request by chri2547**
This PR takes previously refactored thresholds docs and combines them into a single topic.

### CODE REVIEW

File: docs/sources/panels/configure-thresholds/_index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
++++
+title = "Configure thresholds in a Grafana dasboard"
+aliases = ["/docs/grafana/latest/panels/thresholds/", "/docs/grafana/latest/panels/", "/docs/grafana/latest/panels/specify-thresholds/about-thresholds/", "/docs/grafana/latest/panels/specify-thresholds/add-a-threshold/", "/docs/grafana/latest/panels/specify-thresholds/add-threshold-to-graph/", "/docs/grafana/latest/panels/specify-thresholds/delete-a-threshold/"]
+weight = 300
++++
+
+# Configure thresholds in a dashboard
+
+This section includes information about using thresholds in your visualizations.
+
```
Comment (GrafanaWriter) [inline docs/sources/panels/configure-thresholds/_index.md:10]: @jdbaldry - example of where we'd like to insert a TOC for the page. Not a blocker for this PR, just alerting you to the example.

File: docs/sources/panels/configure-thresholds/_index.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+1. In the **Show thresholds** drop-down, select a threshold display option.
+1. Click **Save**.
+
+## Delete a threshold
+
+Delete a threshold when it is no longer relevant to your business operations. When you delete a threshold, the system removes the threshold from all visualizations that include the threshold.
+
+### Before you begin
+
+- [Add a threshold]({{< relref "#add-a-threshold.md" >}}).
```
Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:62]: This feels unnecessary. A user does not want to delete a threshold if they don't already have one

File: docs/sources/panels/configure-thresholds/_index.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+**To delete a threshold**:
```
Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:64]: repeats section title action

File: docs/sources/panels/configure-thresholds/_index.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+1. Navigate to the panel to which you want to add a threshold.
+1. Click the **Field** tab. (Or **Panel** tab for a graph panel.)
```
Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:67]: There is no Field tab anymore, was removed in v8

File: docs/sources/panels/configure-thresholds/_index.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+1. Click the trash can icon next to the threshold you want to remove.
+1. Click **Save** to save the changes in the dashboard.
```
Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:69]: Do we need to prompt them to save?

File: docs/sources/panels/configure-thresholds/_index.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+   Grafana adds a threshold value and color.
+
+1. Accept the recommendations or edit the threshold.
+   - **Edit color:** To select a color, click the color dot.
+   - **Edit number:** To change the threshold value, click in the field and enter a number.
+1. Select a **Threshold mode**.
+   Threshold mode applies to all thresholds on this panel.
+1. In the **Show thresholds** drop-down, select a threshold display option.
+1. Click **Save**.
```
Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:54]: I would remove the save prompt, they are probably doing more things. This feels unrelated to there current workflow

File: docs/sources/panels/configure-thresholds/_index.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+1. Click **+ Add threshold**.
+
+   Grafana adds a threshold value and color.
+
+1. Accept the recommendations or edit the threshold.
+   - **Edit color:** To select a color, click the color dot.
+   - **Edit number:** To change the threshold value, click in the field and enter a number.
+1. Select a **Threshold mode**.
+   Threshold mode applies to all thresholds on this panel.
+1. In the **Show thresholds** drop-down, select a threshold display option.
```
Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:53]: Show thresholds option is only available when you have the time series visualization.  So would move this to a paragraph "Show thresholds in graphs" is something like that

**Review by torkelo — COMMENTED** [review]
Left a few comments, take them as only my layman suggestions. Personally, I am in favor of a more terse docs style that does not repeat things too much and is more to the point. 

Reading ddog or elastic docs they are A LOT terser than some of our docs, reads so much better and is more scannable / easy to find what your looking for in a shorter article that is more to the point.

**Comment by torkelo** [comment]
After this would be good to combine all these as well, and change title to "Configure value mappings" 
https://grafana.com/docs/grafana/latest/panels/format-data/ 

It would be good to link to value mappings from this doc (in the about section) as an alternative way to configure conditional colors, as value mapping can also be used in a similar way to set conditional colors.

File: docs/sources/panels/configure-thresholds/_index.md (hunk 4 above)
Comment (chri2547) [reply docs/sources/panels/configure-thresholds/_index.md:67]: Done.

File: docs/sources/panels/configure-thresholds/_index.md (hunk 5 above)
Comment (chri2547) [reply docs/sources/panels/configure-thresholds/_index.md:69]: It appears that the user doesn't actually need to save their changes, which I didn't actually test. I can specify a threshold and back out of the Edit view without saving, and the threshold I added remains. i'll remove the Save step.

File: docs/sources/panels/configure-thresholds/_index.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+weight = 300
++++
+
+# Configure thresholds in a dashboard
+
+This section includes information about using thresholds in your visualizations.
+
+## About thresholds
+
+A threshold is a value that you specify for a metric that is visually reflected in a dashboard when the threshold value is met or exceeded.
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/panels/configure-thresholds/_index.md:23]: added these points, thanks.

File: docs/sources/panels/configure-thresholds/_index.md (hunk 3 above)
Comment (chri2547) [reply docs/sources/panels/configure-thresholds/_index.md]: ah, we can talk about this. When there are entries in the before you begin section, it's visually jarring to go from a bulleted list to a numbered list. The stem sentence provides a bit of separation between the two. We could have a style that removes the stem sentence if there are no before you begin items.

File: docs/sources/panels/configure-thresholds/_index.md (hunk 1 above)
Comment (torkelo) [reply docs/sources/panels/configure-thresholds/_index.md:20]: @GrafanaWriter Every page get's an automatic TOC in the right side pane:
Screenshot from 2022-04-22 13-47-47

File: docs/sources/panels/configure-thresholds/_index.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+Thresholds provide one method for you to conditionally style and color your visualizations based on query results. You can apply thresholds to most, but not all, visualizations. For more information about visualizations, refer to [Visualization panels]({{< relref "../../visualizations" >}}).
+
+You can use thresholds to:
+
+- Color lines or areas in the [Graph panel]({{< relref "../../visualizations/graph-panel" >}})
```
Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:29]: This should also link to new time series panel, the old Graph panel is deprecated. We do not need to link to it. 

```suggestion
- Color grid lines or grid ares areas in the [Time series visualization]({{< relref "../../visualizations/time-series" >}})
```

File: docs/sources/panels/configure-thresholds/_index.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+- Color lines in the [Time-series panel]({{< relref "../../visualizations/time-series/graph-color-scheme/#from-thresholds" >}})
+- Color the background or value text in the [Stat panel]({{< relref "../../visualizations/stat-panel" >}})
+- Color the gauge and threshold markers in the [Gauge panel]({{< relref "../../visualizations/gauge-panel" >}})
+- Color markers in the [Geomap panel]({{< relref "../../visualizations/geomap" >}})
+- Color cell text or background in the [Table panel]({{< relref "../../visualizations/table" >}})
+- Define regions and region colors in the [State timeline panel]({{< relref "../../visualizations/state-timeline" >}})
+
+There are two types of thresholds:
```
Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:37]: The visualizations are in the visualization docs category (and not the Panels category), and we refer to them as visualizations in the UI, yet here we refer to visualizations with a "panel" suffix.

File: docs/sources/panels/configure-thresholds/_index.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+
+## Add or delete a threshold
+
+You can add as many thresholds to a panel as you want. Grafana automatically sorts thresholds values from highest to lowest.
+
+Delete a threshold when it is no longer relevant to your business operations. When you delete a threshold, the system removes the threshold from all visualizations that include the threshold.
+
+**Before you begin:**
+
+- [Add a panel to a dashboard]({{< relref "../working-with-panels/add-panel.md" >}})
```
Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:60]: this feels unnecessary, should we also prompt user to "Login", open a dashboard? 

They don't need to add a panel to a dashboard, they likely already have one. 

The sentence before reads  `You can add as many thresholds to a panel as you want`  so it's clear this is in the context of a panel.

File: docs/sources/panels/configure-thresholds/_index.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+
+**To add or delete a threshold**:
+
+1. To add a threshold:
+
+   a. Edit the panel to which you want to add a threshold.
+
+   b. On the panel display options, locate the **Thresholds** section and click **+ Add threshold**.
```
Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:68]:
```suggestion
   b. In the options side pane, locate the **Thresholds** section and click **+ Add threshold**.
```

Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:68]:
```suggestion
   b. In the options side pane, locate the **Thresholds** section and click **+ Add threshold**.
```

File: docs/sources/panels/configure-thresholds/_index.md [hunk 13]
```diff
@@ -0,0 +1,102 @@
...
+
+   c. Selct a threshold color, number, and mode.
```
Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:70]:
```suggestion
   c. Select a threshold color, number, and mode.
```

File: docs/sources/panels/configure-thresholds/_index.md [hunk 14]
```diff
@@ -0,0 +1,102 @@
...
+
+**Before you begin:**
+
+- [Add a panel to a dashboard]({{< relref "../working-with-panels/add-panel.md" >}})
+
+**To add or delete a threshold**:
+
+1. To add a threshold:
+
+   a. Edit the panel to which you want to add a threshold.
```
Comment (torkelo) [inline docs/sources/panels/configure-thresholds/_index.md:66]: Here it also says you need a panel, (see comment above). This step is good, but makes the comment section of adding a panel unnecessary.

**Review by torkelo — APPROVED** [review]
Left some more comments, but looks good now!

File: docs/sources/panels/configure-thresholds/_index.md (hunk 11 above)
Comment (chri2547) [reply docs/sources/panels/configure-thresholds/_index.md:60]: The effect we're trying to achieve here is to help users who come through a Google search and land blindly on the page understand that having a panel to work with is a prerequisite to performing this task and to provide a link.

The decision to include a Before you begin section is nuanced and very much driven by our audience. If it is super obvious to our audience (even those who blindly land on the page) that they are looking at these docs and already have a panel, then yes, we should remove this. Your feedback is appreciated and gets me thinking about how we can use the Before you begin section more strategically. I'm all for removing non-value-added content. But in some cases, a Before you begin section can be very useful.

File: docs/sources/panels/configure-thresholds/_index.md (hunk 10 above)
Comment (chri2547) [reply docs/sources/panels/configure-thresholds/_index.md:37]: corrected this, thanks

File: docs/sources/panels/configure-thresholds/_index.md (hunk 11 above)
Comment (chri2547) [reply docs/sources/panels/configure-thresholds/_index.md:60]: Made this change.

File: docs/sources/panels/configure-thresholds/_index.md (hunk 14 above)
Comment (chri2547) [reply docs/sources/panels/configure-thresholds/_index.md:66]: corrected

File: docs/sources/panels/configure-thresholds/_index.md [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+
+The **Base** value represents minus infinity. It is generally the “good” color.
+
+## Add a threshold
+
+You can add as many thresholds to a panel as you want. Grafana automatically sorts thresholds from highest value to lowest.
+
+### Before you begin
+
+- [Add a panel to a dashboard]({{< relref "../working-with-panels/add-panel.md" >}}).
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/panels/configure-thresholds/_index.md]: corrected.

File: docs/sources/panels/configure-thresholds/_index.md [hunk 16]
```diff
@@ -0,0 +1,96 @@
...
+
+**To add a threshold**:
```
[the remark this answers is not in the document]
Comment (chri2547) [reply docs/sources/panels/configure-thresholds/_index.md]: removed

File: docs/sources/panels/configure-thresholds/_index.md (hunk 3 above)
Comment (chri2547) [reply docs/sources/panels/configure-thresholds/_index.md]: removed this

File: docs/sources/panels/configure-thresholds/_index.md (hunk 7 above)
Comment (chri2547) [reply docs/sources/panels/configure-thresholds/_index.md]: qualified this to point out this is only relevant for time-series viz.
